bin/cask

```diff
@@ -153,18 +166,38 @@ def format_version(version):
     return '.'.join(str(part) for part in version)
 
 
+def get_missing_packages():
+    """Get a report of the missing direct dependencies.
+
+    The report is a human-readable list of the any direct dependencies which are
+    required but not present. The empty string is returned if there are none.
+
+    """
+    process = subprocess.Popen([sys.executable, CASK, "missing"], stdout=subprocess.PIPE)
+    stdout, _ = process.communicate()
+
+    if(process.returncode):
+        raise Exception("Oops")
```

Comment:
I am a little unclear what you want me to do here. Just throw an error? Mostly this code is copied from `get_cask_path` which has no handling for the broken case.